idd/Energy+.idd.in

```diff
@@ -78133,21 +78196,169 @@ ElectricLoadCenter:Distribution,
        \key DirectCurrentWithInverterDCStorage
        \key DirectCurrentWithInverterACStorage
        \default AlternatingCurrent
-  A7,  \field Inverter Object Name
+!  A ,  \field Inverter Object Type
```

Comment:
The commented lines should be removed before merging into develop.  I realize they are there for future use.